doc/classes/ImporterMesh.xml

```diff
@@ -46,10 +46,12 @@
 			<return type="void" />
 			<param index="0" name="normal_merge_angle" type="float" />
 			<param index="1" name="normal_split_angle" type="float" />
-			<param index="2" name="bone_transform_array" type="Array" />
+			<param index="2" name="normal_weight" type="float" />
+			<param index="3" name="bone_transform_array" type="Array" />
```

Comment:
You need to bind this in the CPP file as well, also you should put this at the end to not break compatibility, and it should have a default value

Reply:
Thanks - somehow I didn't realize this is script-bound, not just internal documentation. Fixed.

Reply:
Hmm - that doesn't seem to fix .NET bindings (see https://github.com/godotengine/godot/actions/runs/7226635682/job/19692661857?pr=86212). Not sure what the protocol is for cases like this. One option is I can keep the parameter out of the script API for now (e.g. add `generate_lods_impl`, and keep `generate_lods` as is)?

Reply:
I've removed the extra parameter from the scripting API for now to avoid compatibility issues altogether; the extra benefit is that the signature with normal weight at the end is fairly weird, and if we add more parameters here in the future, we might want to adjust the script API to take a dictionary of parameters anyway...

Reply:
You'd need to add compatibility binds, but I think we can go for this, will see what reviewers say